carddrawer/src/test/java/com/meli/android/carddrawer/configuration/DefaultCardTest.kt

```diff
@@ -2,14 +2,14 @@ package com.meli.android.carddrawer.configuration
 
 import androidx.core.content.ContextCompat
 import com.meli.android.carddrawer.R
-import com.meli.android.carddrawer.configuration.base.ConfigurationTestBase
+import com.meli.android.carddrawer.BaseTest
 import com.meli.android.carddrawer.model.CardAnimationType
 import io.mockk.every
 import org.junit.Assert.assertEquals
 import org.junit.Before
 import org.junit.Test
 
-class DefaultCardConfigurationTest: ConfigurationTestBase() {
+class DefaultCardTest: BaseTest() {
```

Comment:
Why we changed the name of class here too?

Reply:
adjusted